cea/plots/__init__.py

```diff
@@ -63,20 +63,34 @@ def dashboard_yml_path(config):
     return dashboard_yml
 
 
-def default_dashboard(config, cache, name='Default Dashboard', description=''):
+def default_dashboard(config, cache, name='Default Dashboard', description='', layout='default'):
```

Comment:
i'm wondering, do we really need 'default' as a layout type? can we not just have 'map' and 'rows'? and just set that in `default_dashboard` as the default parameter value? (then you don't need extra logic to handle the 'default' layout)

Reply:
yeah makes sense. Because I originally wanted to name the original layout as 'default' but then realized that it did not mean anything / does not describe the layout.

Reply:
good thinking!